backend/src/data/migrations/20220817203631_udemy_course_modules_tables.ts

```diff
@@ -0,0 +1,43 @@
+import { Knex } from 'knex';
+
+enum ColumnName {
+  ID = 'id',
+  CREATED_AT = 'created_at',
+  UPDATED_AT = 'updated_at',
+  COURSE_ID = 'course_id',
+  MODULE_INDEX = 'module_index',
```

Comment:
```suggestion
```
not sure that we need this key. since now all vendors will have this key

Reply:
But this is an index of modules sequence to display. I think it should be in module 🤔

Reply:
we just need to have them in order as on the udemy website. we don't need to keep it in the database